src/main/java/trn/logistics/knapsack/algorithm/Mergesort.java

```diff
@@ -0,0 +1,14 @@
+package trn.logistics.knapsack.algorithm;
+
+import trn.logistics.knapsack.dto.Material;
+
+import java.util.ArrayList;
+import java.util.Collection;
+
+public class Mergesort {
```

Comment:
Unused, also better use an available implementation as it will be more optimized